apps/website/components/button/README.md

```diff
@@ -219,39 +219,84 @@ If your icon button is has no text, we recommend adding the `title="` attribute
 
 ### Solid Buttons
 
-<doc-demo src="/demos/button/solid-css.html" demo="/demos/button/solid-css.html"></doc-demo>
+<doc-demo>
+<<< .vuepress/public/demos/button/solid-css.html
+::: slot demo
+!!!include(.vuepress/public/demos/button/solid-css.html)!!!
+:::
```

Comment:
The ```<<<``` imports the file as a code snippet and is pre-rendered by vuepress at build time.

The `::: slot demo``` tells vuepress to render the code in the `code` slot of the component.

The `!!!include()``` imports and inlines the html into the slot rendering the html at build time. Kind of like a Jekyll includes.